src/SearchResults.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import HeadingRow from "./HeadingRow";
+import BodyRow from "./BodyRow";
+import CustomerProfile from "./CustomerProfile";
+import moment from "moment";
+
+const SearchResults = ({ results, setBooking }) => {
+  const [customerId, setCustomerId] = useState(null);
+  const [isSorted, setIsSorted] = useState({
+    title: false,
+    firstName: false,
```

Comment:
You could even initialise `isSorted` as an empty object and your `if` condition on line 26 would still work. The `isSorted[key]` object property would be undefined and `undefined` is a falsy value.